pydrive2/auth.py

```diff
@@ -663,19 +617,10 @@ def Authorize(self):
                 "No valid credentials provided to authorize"
             )
 
-        if self.http is None:
-            self.http = self._build_http()
-        self.http = self.credentials.authorize(self.http)
-        self.service = build(
-            "drive", "v2", http=self.http, cache_discovery=False
-        )
-
     def Get_Http_Object(self):
         """Create and authorize an httplib2.Http object. Necessary for
         thread-safety.
         :return: The http object to be used in each call.
         :rtype: httplib2.Http
         """
-        http = self._build_http()
```

Comment:
Do you still use _build_http()?

Do you still ` Authorize` from the above?

Reply:
Actually, we might not need the Authorized method anymore as I am now using the `AuthorizedHttp` object in google-auth library with handles the adding of credential headers (see this [link](https://github.com/googleapis/google-auth-library-python/blob/main/google/auth/transport/urllib3.py#L203))

Reply:
Do you think I should also remove the `Authorize` method since this is now handled by the new google auth library?

Reply:
I think so, yes. If it's not called from anywhere anymore, right?

We might want to double check things like https://docs.iterative.ai/PyDrive2/oauth/#building-your-own-authentication-flow